_posts/blog/2017-07-19-Actions-for-android-devs.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Google Assistant Development for Android Devs (Part 1)"
+author: Patrick Jackson
+categories: Development
+tags: [Google Assistant, Actions on Google, conversational interfaces, Android, Kotlin]
+---
+
+
+# Google Assistant Development for Android Devs (Part 1)
+
+![](https://developers.google.com/actions/images/surfaces-hero.svg)
+
+There is a lot of buzz and excitment around the [Google Assistant Actions](https://developers.google.com/actions/), and with good reason:  this is an entirely new platform for building experiences for a number of devices.  Building a conversational Action with Google Assistant means it will be available on Android, iOS, Google Home, and others as they com available.
+
+
+For the uninitiated, Actions on Google are third party "Actions" that can be built using Google's APIs.  Actions extend what the Google Assistant can do by letting user talk to them.  They can be accessed with trigger words such as 'Ok Google, talk to Ticketmaster', or 'Ok Google, ask ticketmaster find Yankee's games this weekend' (Ticketmaster Action coming soon!)
+
+<img src="https://storage.googleapis.com/banjotms.appspot.com/Screenshot_20170718-103901.png" width="300"> 
+
+At Ticketmaster Mobile Studio, we decided to use the languages and dev environment we are most familiar with - Kotlin/Java, IntelliJ, & Gradle.  This allows us to use the language and tools we are familiar with, and to reuse some code with Android apps.  However, the documentation and examples do not provide any Java/Kotlin.  Google provides a javascript SDK for building webhooks, however so far there is not an official Java SDK.  At the end of the day this is just json in and json out, so of course we can use any language we choose.
```

Comment:
- Maybe reword one of the first 2 sentences, or just replace one "familiar with" with "comfortable with" to avoid repetition.
- Avoid repetition of "however": "**Unfortunately,** the documentation and examples..."